### PR TITLE
docs(debug.md): Added documentation file

### PR DESCRIPTION
**Pull request by SuelenKarbivnychyy**
Developed instructions to proper set up VS Code IDE to easily debug the project.

This Pr closes the issue #74

### CODE REVIEW

**Comment by SuelenKarbivnychyy** [comment]
@sansyrox Please review :)

File: debug.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+To debug this project in VS Code we should select python interpreter from virtual environment created by poetry.
```
Comment (sansyrox) [inline debug.md:1]: One suggestion - Can we add a heading?

File: debug.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		Executable:     /home/suelenk/.cache/pypoetry/virtualenvs/starfyre--dCuxLEF-py3.10/bin/python
+		Valid:          True
+
+		System
+		Platform:   linux
+		OS:         posix
+		Python:     3.10.6
+		Path:       /usr
+		Executable: /usr/bin/python3.10
+
```
Comment (sansyrox) [inline debug.md:24]: Let's convert these into bullets

File: debug.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+3. Copy `file-path` output from `Executable`
+4. In VSCode press Ctrl + Shift + P, type `Python env` in search box, select `Python:Create Environment``
+	select `venv``
+	click on `"+ Enter Interpreter path"`
+	enter file-path output from step #3 to python executable and press enter
```
Comment (sansyrox) [inline debug.md:29]: same for this

File: debug.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+		System
+		Platform:   linux
+		OS:         posix
+		Python:     3.10.6
+		Path:       /usr
+		Executable: /usr/bin/python3.10
+
+3. Copy `file-path` output from `Executable`
+4. In VSCode press Ctrl + Shift + P, type `Python env` in search box, select `Python:Create Environment``
```
Comment (sansyrox) [inline debug.md:26]:
```suggestion
4. In VSCode press `Ctrl + Shift + P`, type `Python env` in the search box, and select `Python: Create Environment``
```

File: debug.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            "justMyCode": true,
+            "args": ["--path",
+            "test-application/",
+            "--build"
+            ],
+            "stopOnEntry": false,
+            "console": "integratedTerminal",
+        }
+    ]
+}
```
Comment (sansyrox) [inline debug.md:56]: Let's wrap this in a code block, i.e. three backticks

**Review by sansyrox — COMMENTED** [review]
Hey @SuelenKarbivnychyy 👋 

Thank you for the PR 😄 

Can we move this file to the `docs/debug.md` path?

Please do let me know if you require any more information from me 😄

File: debug.md (hunk 1 above)
Comment (SuelenKarbivnychyy) [reply debug.md:1]: Yes

File: debug.md (hunk 2 above)
Comment (SuelenKarbivnychyy) [reply debug.md:24]: Done :)

**Comment by SuelenKarbivnychyy** [comment]
Hey, @sansyrox :)

Thank you for the suggestions. 🤗

I have done some changes.

Please review 🤗

File: poetry.lock
Comment (sansyrox) [inline poetry.lock:1]: This is not needed in this PR 😄

File: pyproject.toml
Comment (sansyrox) [inline pyproject.toml:1]: This too.

File: poetry.lock
Comment (sansyrox) [inline poetry.lock:1]: Hey @SuelenKarbivnychyy , you ended up removing the entire `poetry.lock` and `pyproject`. We don't want that 😅

File: docs/debug.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+### To debug this project in VS Code need to select python interpreter from virtual environment created by poetry.
```
Comment (sansyrox) [inline docs/debug.md:1]: @SuelenKarbivnychyy 👋 

This should not be the heading as this is the first line of the debugging documentation. We should have a more concise heading

File: debug.md
Comment (sansyrox) [inline debug.md:1]: we can get rid of this debug.md and just have on in the docs

Comment (SuelenKarbivnychyy) [reply debug.md:1]: > we can get rid of this debug.md and just have on in the docs

What do you mean? Should I add it to README.md ?

File: docs/debug.md (hunk 6 above)
Comment (SuelenKarbivnychyy) [reply docs/debug.md:1]: Thank you, I will address it :)

**Pull request closed without merging**